107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -111,13 +111,13 @@
 
       CacheConfiguration<K, V> cacheConfiguration = builder.build();
 
-      if (!useJsr107Expiry) {
+      if (hasConfiguredExpiry) {
```

Comment:
Because a configured expiry means we should not use the JSR-107 provided one. This stems from changing method name (and contract) on the builder: `CacheConfigurationBuilder.hasConfiguredExpiry()` vs the former `CacheConfigurationBuilder.hasDefaultExpiry`
When I was writing the javadoc, expressing the contract with default name was really cumbersome and felt way more natural with changing the name and meaning of `true` and `false`.